source/docs/03-metadata.xml

```diff
@@ -1338,7 +1339,9 @@
                   </specList>
                </p>
                <p>Many of these elements are already described in chapter <ptr target="#headerstructure"/>, especially in <ptr target="#headerWorkDescription"/>.</p>
-               <p>The <gi scheme="MEI">manifestationList</gi> is available to create lists of physical sources representing a work, for instance for use in a thematic catalog or a critical edition. The <gi scheme="MEI">manifestation</gi> child element corresponds to the <ptr target="#FRBR"/> level of the same name, that is, it describes embodiments of certain expressions of a work. The list below reflects the order in which the optional components of manifestation must occur. <specList>
+               <p>The <gi scheme="MEI">manifestationList</gi> is available to create lists of physical sources representing a work, for instance for use in a thematic catalog or a critical edition. The <gi scheme="MEI">manifestation</gi> child element corresponds to the <ptr target="#FRBR"/> level of the same name, that is, it describes embodiments of certain expressions of a work. The list below reflects the order in which the optional components of manifestation had to occur before MEI 6.0.
```

Comment:
@riedde @pe-ro Same here. Short indication of how it should/can be used now (with MEI 6.0 and later) would be helpful and reduce confusion.

Reply:
I thought about cutting this text, but then we will lost the documentation for pre-MEI6. What about something like: "The following elements can be used in any order. Before MEI6 they had to be ordered as shown below"?

Reply:
Again, this is the documentation *for* 6.0, not the documentation for earlier versions.